### PR TITLE
Add changelog notes

### PR DESCRIPTION
**Pull request by psergee**
Added changelog note about the fix for "Failed to dial" errors.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   * option --update has been added - sets percentage of update operations in bench space.
   * option --fill" has been added - sets number of records to pre-fill the space.
 
+### Fixed
+
+- Fixed sporadic "Failed to dial" errors on cartridge cli different commands (#697).
```
Comment (LeonidVas) [inline CHANGELOG.md:20]: This is for releasenotes, you don't need a ticket number. You can add a clarification that the connection to the local instance will now be used.

**Review by LeonidVas — CHANGES_REQUESTED** [review]
Hi! Thank you for the patch.
Commit title doesn't match the code style (`documentation: ...`).
In commit buddy `Follow-ups #...`

File: CHANGELOG.md (hunk 1 above)
Comment (psergee) [reply CHANGELOG.md]: Done.

**Review by LeonidVas — APPROVED** [review]
LGTM

**Pull request merged**